packages/govuk-frontend/src/govuk/components/accordion/template.njk

```diff
@@ -35,30 +35,32 @@
   }) -}}
 
   {%- if params.rememberExpanded !== undefined %} data-remember-expanded="{{ params.rememberExpanded | escape }}"{% endif %}
-  {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+  {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
   {% for item in params.items %}
-    {% if item %}
-      <div class="govuk-accordion__section {%- if item.expanded %} govuk-accordion__section--expanded{% endif %}">
-        <div class="govuk-accordion__section-header">
-          <h{{ headingLevel }} class="govuk-accordion__section-heading">
-            <span class="govuk-accordion__section-button" id="{{ id }}-heading-{{ loop.index }}">
-              {{ item.heading.html | safe if item.heading.html else item.heading.text }}
-            </span>
-          </h{{ headingLevel }}>
-          {% if item.summary.html or item.summary.text %}
-            <div class="govuk-accordion__section-summary govuk-body" id="{{ id }}-summary-{{ loop.index }}">
-              {{ item.summary.html | safe if item.summary.html else item.summary.text }}
-            </div>
-          {% endif %}
-        </div>
-        <div id="{{ id }}-content-{{ loop.index }}" class="govuk-accordion__section-content" aria-labelledby="{{ id }}-heading-{{ loop.index }}">
-          {% if item.content.html %}
-            {{ item.content.html | safe }}
-          {% elif item.content.text %}
-            <p class="govuk-body">{{ item.content.text }}</p>
-          {% endif %}
-        </div>
+  {% if item %}
```

Comment:
@owenatgov Instead of indenting more (to add extra spaces to the HTML) how about un-indenting?

Just pushed, here's Accordion:

https://github.com/alphagov/govuk-frontend/compare/7a8beb4dd425acf91c07389b2d9e847d15e91966..196ba655eab63dc9ade9178eff00f63afcf0542c#diff-0b0d5366e0718a4d3960cf2f8a280eede3edf141491d8d99c5dc9cdc39a3f4e5